foundry/applications/formApplications/baseEntitySheets/actorSheet.d.ts

```diff
@@ -153,4 +153,34 @@ declare namespace ActorSheet {
     data: BaseEntitySheet.Data<O>['entity']['data'];
     items: BaseEntitySheet.Data<O>['entity']['items'];
   }
+
+  namespace DropData {
+    type Combined = ActiveEffect | Actor | Item | Folder;
```

Comment:
Can we type this as interface instead? That would allow people to add to it more easily by using declaration merging.

Reply:
How would you do this though? Just merging them together? That would take away the possibility to distinguish what you have by the `type` property, which Foundry actually uses.

Reply:
Ah, true... it’s ok I guess

Reply:
Since that was the last thing in this review to resolve, can you approve it? The remaining conversations where the same.